services/dynamic/dynamic-xml.service.js

```diff
@@ -10,6 +10,20 @@ import {
 } from '../index.js'
 import { createRoute } from './dynamic-helpers.js'
 
+const description = `
+The Dynamic XML Badge allows you to extract an arbitrary value from any
+XML Document using an XPath selector and show it on a badge.
+
+Useful resources for constructing XPath selectors:
+- [XPather](http://xpather.com/)
+- [XPath Cheat Sheet](https://devhints.io/xpath/)
+
+Note: For XML documents that use a default namespace prefix, you will need to use the
+[local-name](https://developer.mozilla.org/en-US/docs/Web/XPath/Functions/local-name)
+function to construct your query.
+For example \`/*[local-name()='myelement']\` rather than \`/myelement\`.
```

Comment:
@Washingtonwei - a question for you here: If this had been in the documentation on the website, do you think this would have been enough to allow you to work out https://github.com/badges/shields/issues/9784 without raising an issue?

If not, can you suggest any improvements that don't relate to a specific XML document?

Reply:
This is perfect! Thank you!